From 5f7c31cf238d50a73ed1f1941ddc65c21fee2113 Mon Sep 17 00:00:00 2001
From: Po Liu <po.liu@nxp.com>
Date: Fri, 1 Jul 2016 15:45:36 +0800
Subject: [PATCH 008/124] fdt/ls2088: correct the ls2088 pci aer interrupt name

For layerscape, since not support MSIX/MSI/INTA interrupt for
aer. It only support the standalone error interrupt. So add
interrupt name 'aer' in the fdt file.

This patch to correct the ls2088 fdt file in pci part.

Signed-off-by: Po Liu <po.liu@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
index eac98f2..d197af6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
@@ -443,7 +443,7 @@
 		       0x20 0x00000000 0x0 0x00001000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 108 0x4>; /* Level high type */
-		interrupt-names = "intr";
+		interrupt-names = "aer";
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
@@ -468,7 +468,7 @@
 		       0x28 0x00000000 0x0 0x00001000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 113 0x4>; /* Level high type */
-		interrupt-names = "intr";
+		interrupt-names = "aer";
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
@@ -493,7 +493,7 @@
 		       0x30 0x00000000 0x0 0x00001000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 118 0x4>; /* Level high type */
-		interrupt-names = "intr";
+		interrupt-names = "aer";
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
@@ -518,7 +518,7 @@
 		       0x38 0x00000000 0x0 0x00001000>; /* configuration space */
 		reg-names = "regs", "config";
 		interrupts = <0 123 0x4>; /* Level high type */
-		interrupt-names = "intr";
+		interrupt-names = "aer";
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
-- 
2.1.0.27.g96db324

